### DISCUSSION TITLE
Reload also the overlay and new function for my routes

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#665] [Q&A] [answered]
The standard behavior is that the route is loaded again, what is good, when the app is exited and opened again.
When I loaded a Tour including the overlay and exit and reopen the app or when the app crashes (seldomly), then I also want to have the overlay reloaded, but only the route is reloaded. 

Can there be a check if the overlay was loaded before then also the overlay should be reloaded together with the route.

And can there be an additional function when I load the tour from my routes via swipe right (and also via context menu "load route" with additional question), best intuitive via long tap and swipe right (in case this is an own trigger), that also die overlay is loaded together with the route?

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom Thanks for the suggestions.

These are different things and were discussed before:
- https://github.com/devemux86/cruiser/discussions/431
- https://github.com/devemux86/cruiser/discussions/646

The route and track overlays are managed in My collection.
We can set their visibility, zoom level, color, etc. with many options for a custom appearance.

The menu in "Routes" has separate actions "Show on map" and "Load route", no need to mix them.
We can think if the swipe right could show a selection dialog for these actions, as a faster method.

**Comment by DolomitesTom** [comment]
>The route and track overlays are managed in My collection.
We can set their visibility, zoom level, color, etc. with many options for a custom appearance.

Yes ok, but you can't just quickly delete these overlays by one function and it is difficult to find the right ones in myroutes to make them unvisible. This needs to be improved in my point of view, maybe by long-press on the overlay and then show the right context menu to make exactly this unvisible or change zoom, color for exactly this which you want to change on the map.

>We can think if the swipe right could show a selection dialog for these actions, as a faster method.
This would help.
